include/misc/rb.h

```diff
@@ -159,8 +159,9 @@ struct _rb_foreach {
 }
 #else
 #define _RB_FOREACH_INIT(tree, node) {					\
-	.stack   = alloca((tree)->max_depth * sizeof(struct rbnode *)), \
-	.is_left = alloca((tree)->max_depth * sizeof(char)),		\
+	.stack   = (struct rbnode **)					\
+					alloca((tree)->max_depth * sizeof(struct rbnode *)), \
```

Comment:
Nitpick: what's up with the whitespace here?  Looks like you broke the line for length, but then indented it farther than it already was?

Reply:
You're right. I modified the file for reduce indentation.

Reply:
You're right. I modified the file for reduce indentation.